omod/pom.xml

```diff
@@ -79,6 +79,25 @@
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.thymeleaf</groupId>
+			<artifactId>thymeleaf</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.github.ben-manes.caffeine</groupId>
+			<artifactId>caffeine</artifactId>
+		</dependency>
+<!--	///	Just for testing ///    -->
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.8.5</version>
+		</dependency>
+<!--	////////////////////////    -->
```

Comment:
Several things:

* Comments here are not very helpful. If we need a resource just for testing, it **must** to be included in the base POM with the `<scope>test</scope>`. Otherwise, this jar ends up bundled in the OMOD (by default, Maven artifacts are loaded with the scope `compile` unless you tell Maven otherwise).
* We should move away from using GSON at all. HAPI has moved from GSON to using Jackson, which should already be available on the class path.

Reply:
@ibacher Actually I did not mean it that way. I meant that it won't actually be a part of the final commit. 

Also, for the `print` statements, I do not intend to keep it as a part of the final commit. I just used it to see the output for now. I've used `assert` statements in the tests which would be a part of the final commit.

But yes, I'll remove such code which I don't plan to keep in the final commit, i.e. the `google-gson` dependency and the `print` statements. Or maybe if I need it at all, I'll use Jackson as you suggested.

Reply:
Ah! Ok... I see now!